src/SchemaUtils.elm

```diff
@@ -934,6 +992,34 @@ typeToDecoder qualify type_ =
         Common.String ->
             CliMonad.succeed Gen.Json.Decode.string
 
+        Common.DateTime ->
+            CliMonad.succeed
+                (Gen.Json.Decode.string
+                    |> Gen.Json.Decode.andThen
+                        (\raw ->
+                            Gen.Result.caseOf_.result
+                                (Gen.Parser.Advanced.call_.run Gen.Rfc3339.dateTimeOffsetParser raw)
+                                { ok = \record -> Gen.Json.Decode.succeed (Elm.get "instant" record)
+
+                                -- TODO: improve error message
+                                , err = \_ -> Gen.Json.Decode.fail "Invalid date"
```

Comment:
```suggestion
                                , err = \_ -> Gen.Json.Decode.fail "Invalid RFC-3339 date-time"
```
Not sure sure if this is better or not

Reply:
I like it's at least better than "Invalid date"